doc/lsst.ap.verify/running.rst

```diff
@@ -0,0 +1,80 @@
+.. _ap-verify-running:
+
+#######################################
+Running ap_verify From the Command Line
+#######################################
+
+``ap_verify`` is a Python script designed to be run on both developer machines and verification servers.
+While ``ap_verify`` is not a :ref:`command-line task<lsst.pipe.base>`, the command-line interface is designed to resemble that of command-line tasks where practical.
+This page describes the minimum options needed to run ``ap_verify``.
+For more details, see the :ref:`ap-verify-cmd` or run :option:`ap_verify.py -h`.
+
+.. _ap-verify-dataset-name:
+
+Datasets as Input Arguments
+---------------------------
+
+Since ``ap_verify`` begins with an uningested :ref:`dataset<ap-verify-datasets>`, the input argument is a dataset name rather than a repository.
+
+.. TODO: is the abstraction of having a separate `dataset` name useful, or does it just impose more work on the user (who needs the repo name anyway to install the dataset)? (DM-12853)
+
+Datasets are identified by a name that gets mapped to an :ref:`eups-registered directory <ap-verify-datasets-install>` containing the data.
+The mapping is :ref:`configurable<ap-verify-configuration-dataset>`.
+
+.. _ap-verify-run-output:
+
+How to Run ap_verify in a New Workspace
+---------------------------------------
+
+Using the :ref:`HiTS 2015 <ap_verify_hits2015-package>` dataset as an example, one can run ``ap_verify`` as follows:
+
+.. TODO: why dataIdString instead of id or dataId? (DM-12853)
+
+.. prompt:: bash
+
+   python ap_verify/bin/ap_verify.py --dataset HiTS2015 --output workspace/hits/ --dataIdString "visit=54123 ccd=25 filter=g" --silent
```

Comment:
I would note here that running `ap_verify` in parallel causes the metric output to be clobbered, and reference DM-13042 (which is the ticket to fix that).

Reply:
Are you sure? Documentation doesn't normally mention bugs unless they're not going to be fixed for a while, and this one seems pretty straightforward.